Component removal and HasNot fix (#13)
flame-engine/oxygen: CHANGELOG.md

## 0.3.0

> - Thanks to Bill Haggerty for their contribution of adding proper component removal

Reviewer: ```suggestion
```

imho this should not be in a CHANGELOG, while we do appreciate each and every contributor we like to have the GIT commits and their authors speak for themselves.

> - Fixed HasNot filters to properly filter after a filtered component is added

Reviewer: ```suggestion
- Fixed `HasNot` filters to properly filter after a filtered component is added
```

> - Bumped min dart version to 2.15.0 to allow for constructor tear-offs

Reviewer: ```suggestion
- **BREAKING**: Bumped Dart to v2.15.0 to allow for constructor tear-offs
```

> - No breaking changes

Reviewer: ```suggestion
```
